music21/metadata/properties.py

```diff
@@ -1175,6 +1215,15 @@ class PropertyDescription:
         if x.oldMusic21WorkId
 }
 
+UNIQUE_NAME_TO_MUSIC21_WORK_ID: t.Dict[str, str] = {
+    x.uniqueName if x.uniqueName
+    else x.name:
+        x.oldMusic21WorkId if x.oldMusic21WorkId
+        else ''
+        for x in STANDARD_PROPERTY_DESCRIPTIONS
+        if x.oldMusic21WorkId
+}
```

Comment:
please rewrite as a standard loop.  I can't figure out what `if` goes to what.  Only singly-nested generator functions appear (or should appear) in music21 code and this is triply nested.

Reply:
Aw man, I was so proud of writing that crazy thing.  I can't read it either, though, so yeah I'll redo those.

Reply:
it was indeed remarkably impressive.  the part that made me say, "nah, as amazing as this is, it's gotta go" was seeing `else x.name:` with a line break after it, and I was like, "wait a second, else can't take an expression after it, he's thinking of elif..." before I realized that the `:` was the separator of the `key:value` of the dictionary.

Reply:
Done.